src/main/json2xlsx.js

```diff
@@ -4,15 +4,17 @@ const assert = require('assert');
 const fileUtils = require('./lib/file-utils');
 const ccdUtils = require('./lib/ccd-spreadsheet-utils');
 const stringUtils = require('./lib/string-utils');
-const { Substitutor } = require('./lib/substitutor');
+const sheetUtils = require('./lib/sheet-utils');
+const {Substitutor} = require('./lib/substitutor');
 
 const sourceXlsx = './data/ccd-template.xlsx';
 
 const validateArgs = (args) => {
   assert(!!args.sheetsDir, 'sheets directory argument (-D) is required');
   assert(!!args.destinationXlsx, 'spreadsheet file argument (-o) is required');
 
-  assert(fileUtils.exists(args.sheetsDir), `sheets directory ${args.sheetsDir} not found`);
+  assert(fileUtils.exists(args.sheetsDir),
```

Comment:
Please revert this. It adds no value. It does not make it more readable. Original line length was within acceptable threshold.

Reply:
Missed that in auto formatting issues. Originally the line length was set to 80, now 120